### DISCUSSION TITLE
ldap login method

### DISCUSSION THREAD

**Discussion by 2Ppisa** [quarkiverse/quarkus-vault#300] [General]
Hi all,
[ldap login method ](https://developer.hashicorp.com/vault/api-docs/auth/ldap#login-with-ldap-user) doesnt's seem to be supported is there some particular reason, about?

Thanks in advance.
Paolo

**Comment by kdubb** [comment]
@2Ppisa No reason other than it seems it's never been requested. PR's are welcome.
